markhor_slam/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>markhor_slam</name>
+  <version>0.0.0</version>
+  <description>The markhor_slam package</description>
+
+  <maintainer email="capra@clubcapra.com">charles</maintainer>
+
+  <license>MIT</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
```

Comment:
You should add that the package depends on `hector_mapping` and `sick_tim` this allows the aromatization of installing the package like we did for you when you started working on this task.